resources/github-actions.md

```diff
@@ -204,14 +204,18 @@ RMM_CHANNEL=$(rapids-get-pr-conda-artifact rmm 1223 python)
 LIBKVIKIO_CHANNEL=$(rapids-get-pr-conda-artifact kvikio 224 cpp)
 
 # Install library packages with the CI artifact channels providing the updated dependencies for testing
-
-rapids-mamba-retry install \
-  --channel "${CPP_CHANNEL}" \
-  --channel "${PYTHON_CHANNEL}" \
-  --channel "${LIBRMM_CHANNEL}" \
-  --channel "${LIBKVIKIO_CHANNEL}" \
-  --channel "${RMM_CHANNEL}" \
-  cudf libcudf
+# ...
+rapids-dependency-file-generator \
+  --output conda \
+  --file-key "${FILE_KEY}" \
```

Comment:
Do we want to explain how this connects to the `dependencies.yaml` key?

Think the detail about this key including the packages produced in the previous build step would be interesting to readers

Reply:
Done in f6156f3.